addons/binding/org.openhab.binding.milight/src/main/java/org/openhab/binding/milight/internal/protocol/MilightDiscover.java

```diff
@@ -76,62 +84,62 @@ public MilightDiscover(DiscoverResult discoverResult, int resendTimeoutInMillis,
         this.discoverResult = discoverResult;
     }
 
+    /**
+     * Closes the socket and waits for the thread to shutdown.
+     * You cannot reuse this object after calling dispose.
```

Comment:
So this is not the `dispose` required by framework? That is what I missed :-)

Better not call it `dispose` then because we use that a lot in the framework with different meaning. Maybe `destroy` suits your needs it sounds like it demolishes something, if I destroy something I would not try to use it after that :-D